lib/io.js

```diff
@@ -303,3 +312,7 @@ function randomBytes (n) {
   sodium.randombytes_buf(buf)
   return buf
 }
+
+function encodeIP (peer) {
```

Comment:
can we improve on this workaround in future some way without sacrificing on latency? For instance, can we store both hostname and IP of bootstrap nodes to side step it

Reply:
We should just tag the bootstrap nodes with bootstrap: true when we add them initially to the routing table (PR welcome :))